Sources/Turf/WKT.swift

```diff
@@ -0,0 +1,222 @@
+import Foundation
+
+/**
+ Entity which can be encoded to and decoded from 'Well Known Text'.
+ */
+public protocol WKTCodable {
```

Comment:
The name of this protocol implies that it’s somehow related to Codable, but it doesn’t appear to be. Instead, it looks very similar to protocols such as GeometryConvertible and GeoJSONObjectConvertible. Consider renaming this protocol to WKTConvertible.